.github/workflows/installer.yml

```diff
@@ -0,0 +1,115 @@
+name: installer
+
+on:
+  push:
+    paths:
+      - build/*
+      - .github/*
+  pull_request:
+    paths:
+      - build/*
+      - .github/*
+  release:
+defaults:
+  run:
+    shell: pwsh
+env:
+  SCRIPT_URL: "https://raw.githubusercontent.com/QutEcoacoustics/audio-analysis/${{ github.sha }}/build/download_ap.ps1"
+  GITHUB_AUTH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+jobs:
+  test-script-host:
+    runs-on: ubuntu-latest
+    steps:
+      - name: ensure script host serves the script with "charset=utf-8"
+        run: |
+          curl -Ii "$env:SCRIPT_URL" | Write-Output -OutVariable "response"
+          $response | grep "charset=utf-8"
+      - name: ensure short url redirects to download script
+        # if this test ever fails, recreate short url and update the installing.md docs
+        run: |
+          curl -Ii "https://git.io/JtOo3"  | Write-Output -OutVariable "response"
+          $response | grep "https://raw.githubusercontent.com/QutEcoacoustics/audio-analysis/.*/build/download_ap.ps1"
+
+  test-installer:
+    strategy:
+      fail-fast: false
+      matrix:
+        os:
+          - windows-latest
+          - ubuntu-latest
```

Comment:
I would actually like to see this run on multiple versions of linux if possible. People/companies have a habit of sitting 1 or 2 versions behind the latest version of ubuntu because of a number of valid reasons. If we support those versions, we should add them to the tests

Reply:
we're limited to what GA offers but I'll add more variants